### ISSUE TITLE
question about the paper and model

### ISSUE THREAD

**Issue by ppdk-data** [suyukun666/S2CNet#4]
Hello, i have a question about the paper and model.

To generate the cropping result of the aspect ratio, it is necessary to create several candidate boxes by myself? 
The model's inputs is candidate boxes, and the outputs is the score?

**Comment by ppdk-data** [comment]
because the final output is mos, not the subimage. Is that correct?

**Comment by suyukun666** [comment]
> Hello, i have a question about the paper and model.
> 
> To generate the cropping result of the aspect ratio, it is necessary to create several candidate boxes by myself? The model's inputs is candidate boxes, and the outputs is the score?

Yes. you can refer to like：https://github.com/suyukun666/S2CNet/blob/main/data/croppingdataset.py#L204 generate the specific ratio boxes
